### ISSUE TITLE
Poor quality of stations in AAC and AAC +

### ISSUE THREAD

**Issue by hevet** [Edzelf/ESP32-Radio#476]
When I turn on a station in AAC or AAC + format, for a split second you hear good quality, and then the quality drops and you hear that it is in mono. This is not a problem with MP3. On the computer, these stations can be heard in stereo and in good quality. Where is the problem?

**Comment by Edzelf** [comment]
Always give an example of the station that gives the problem.  Maybe someone can try to reproduce it.

**Comment by hevet** [comment]
Exemplary two stations, at each station in these formats I have poor quality.

http://0n-disco.radionetz.de/0n-disco.aac
http://live.elka.pl:8000/miedziowe

**Comment by Edzelf** [comment]
Sorry, All 4 stations sound good on my set-up.  Are you sure you have the VS1053 chip on the decoder board?

**Comment by hevet** [comment]
Yes, I have tiles with VS1053B. I checked on two different ones. I found another radio project and there the AAC and AAC + stations sound very good. Unfortunately, the stations are of poor quality in your project. I also checked the stations in OGG and these also sound good. The problem is clearly with AAC and AAC + stations.

**Comment by Rainer-G** [comment]
All stations play without drop-outs, but with a rather low Bitrate_ 64k http://0n-disco.radionetz.de/0n-disco.aac, 64k http://stream2.nadaje.com:8094/ 48k http://uk2.internet-radio.com:8024/stream,48k http://saw-de-hz-fal-stream06-cluster01.radiohost.de/saw-fitness_48?ref=radiosure-1a,  if you replace the 48 with 128, this Station http://saw-de-hz-fal-stream06-cluster01.radiohost.de/saw-fitness_48?ref=radiosure-1a will Play at a higher Bitrate! Winamp shows the same rate so it looks as if it is not a problem of the ESP32 Webradio ? Same Station with different bitrates ***@***.*** : would it be possible to show format and stereo/mono in the Web Interface when sending the Test command? RegardsRainer Gesendet von Mail für Windows Von: JacekGesendet: Sonntag, 29. August 2021 17:41An: Edzelf/ESP32-RadioCc: SubscribedBetreff: Re: [Edzelf/ESP32-Radio] Poor quality of stations in AAC and AAC + (#476) Exemplary two stations, at each station in these formats I have poor quality.http://0n-disco.radionetz.de/0n-disco.aachttp://live.elka.pl:8000/miedziowe—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or unsubscribe.Triage notifications on the go with GitHub Mobile for iOS or Android.
